styleguide/template.md

```diff
@@ -347,13 +347,33 @@ function fancyAlert(arg) {
 
 ## Videos
 
+Currently, OPS can embed both Channel 9 and youtube videos with the following syntax:
+
 ### Channel 9
 
-<iframe src="https://channel9.msdn.com/Shows/On-NET/Shipping-NET-Core-RC2--Tools-Preview-1/player" width="960" height="540" allowFullScreen frameBorder="0"></iframe>
+```markdown
+> [!VIDEO <channel9_video_link>]
+```
+
+To get the video's correct URL, select the **Embed** tab below the video frame, and copy the URL from the `<iframe>` element. For example:
+
+```markdown
+> [!VIDEO https://channel9.msdn.com/Series/A-Nonexistent-Video/03/player]
```

Comment:
I'd keep the link to an existing video. Maybe replace with a newer one such as 
`https://channel9.msdn.com/Blogs/dotnet/NET-Core-20-Released/player`